SMLHelper/Assets/ModPrefab.cs

```diff
@@ -99,15 +116,17 @@ internal GameObject GetGameObjectInternal()
             {
                 pid.ClassId = this.ClassID;
             }
-
-            return go;
         }
 
+
         /// <summary>
         /// Gets the prefab game object. Set up your prefab components here.
         /// The <see cref="TechType"/> and ClassID are already handled.
         /// </summary>
         /// <returns>The game object to be instantiated into a new in-game entity.</returns>
-        public abstract GameObject GetGameObject();
+        public virtual GameObject GetGameObject() => null; // SUBNAUTICA_EXP TODO: remove (or make obsolete) for SN after async update
```

Comment:
What's going on with the TODOs from #181?
Also, this should be done before merge.

Reply:
This one (and others that marked with SUBNAUTICA_EXP TODO) should be fixed after SN async update, it's just a reminder. Others TODOs I'll fix before merge of course, I'm still testing all that on my mods.
I can switch it to draft, but with exceptions of TODOs all code is working now.

Reply:
Could you at least at a little more description to the `TODO` comments?  
Something like this
```CS
// TODO - Remove this block after SN1 Experimental changes are in Stable
```

Reply:
As long as the `TODO` comment explains what we need to do and when, I'm fine with these.

`TODO`s in XML documentation should be updated as soon as they can (assuming we aren't expecting drastic changes to the documentation.

Reply:
After some testing I realized that sync `GetGameObject()` still can be useful in exp branch if it used with asset bundles, for example. If it doesn't require access to the game prefabs, it can work synchronously. So I remove this specific TODO and couple of others. For now, it's just one TODO left, for the `PrefabDatabase.GetPrefabForFilename` patch, that method is removed in the exp branch.